backends/s3/src/main/scala/geotrellis/s3/S3Client.scala

```diff
@@ -14,10 +14,10 @@ import com.amazonaws.ClientConfiguration
 import org.apache.commons.io.IOUtils
 import scala.collection.JavaConversions._
 
-trait S3Client extends LazyLogging {
+trait S3Client extends LazyLogging with Serializable {
```

Comment:
The code in question is here: https://github.com/jamesmcclain/geotrellis/blob/feature/jwm/backends/spark-test/src/test/scala/geotrellis/spark/render/RenderedImageSpec.scala#L23